src/abstractdataframe/sort.jl

```diff
@@ -341,23 +341,33 @@ If `rev` is `true`, reverse sorting is performed. To enable reverse sorting only
 for some columns, pass `order(c, rev=true)` in `cols`, with `c` the
 corresponding column index (see example below).
 
+Since having repeated elements makes multiple sorting orders valid, the
+`checkunique` keyword allows for the situation to be caught. If `checkunique` is
+`true` and duplicate elements are found an `ArgumentError` will be thrown. The
```

Comment:
```suggestion
`true` and duplicate elements are found an error will be thrown. The
```
(typically we avoid specifying error type in the manual to make future maintenance easier.